packages/forms-web-app/src/controllers/rule-6/enter-code.js

```diff
@@ -105,14 +110,13 @@ const postEnterCodeR6 = (views) => {
 
 		const sessionEmail = getSessionEmail(req.session, false);
 
-		const tokenValid = await isTokenValid(
-			token,
-			enterCodeId,
-			sessionEmail,
-			action,
-			req.session?.appeal?.lpaCode,
-			true
-		);
+		const isRule6User = await isRule6UserByEmail(req, sessionEmail);
+
+		if (!isRule6User) {
+			return res.redirect(`/${views.EMAIL_ADDRESS}`);
```

Comment:
put this check after the other token checks and do
return renderError('Enter the correct code');

Reply:
Part of my reasoning for putting it here was to avoid a non-Rule 6. user being able to use the override test code in dev and test.  Any problem with having the check here, before the token checks, but with the renderError rather than redirect?

Reply:
Actually, have put the rule 6 check after some token checks, makes sense and still prevents non-r6 using the test code, it occurs before the rule6 user session creation

Reply:
No problem think I was thinking through a way to get the tooManyAttempts error to still work for non rule 6 users, but that isn't really possible

yes main thing here is to render error not redirect